server/python_backend/gmail_routes.py

```
@@ -0,0 +1,207 @@
+        # Background performance monitoring
+        background_tasks.add_task(
+            performance_monitor.record_sync_performance,
+            result
+        )
+
+        return result
+    except GoogleApiHttpError as gmail_err:
+        error_details_dict = {}
+        try:
+            error_details_dict = json.loads(gmail_err.content.decode())
+        except: # Broad except for decoding issues
```

Ruff (0.11.9)

Do not use bare `except`

(E722)